Topics/Product_Management.md

```diff
@@ -12,3 +12,45 @@ Some key features of a successful product manager:
+* Irene’s team was able to react swiftly and effectively to the changes brought on by COVID-19 by first reviewing an existing product roadmap they had, which allowed everyone on the team to have a common document for reference as they made their decisions and reviewed timelines.
+
+Irene sees the pandemic as transformative, allowing unprecedented work flexibilit. She finds that the newly added option of working remotely adds value to her work and increases the potential of teams. She believes that by incorporating some of these strategies, many product managers will be able mirror her success.
+
+
+#### Case Study 2 - Working Remotely:
+
+We now report the experience of several product managers working at TopTal remotely, referencing this [article](https://www.toptal.com/product-managers/remote/product-managers-shift-to-remote). Sam Nissinen and Darshan Dave, two product managers at TopTal, tell us how this transition presented challenges due to the difficulty of not being physically present with teams. This required product managers to enhance their communication skills but ultimately led to success. 
+
+Sam and Dave, succeeded at overcoming this new challenge by:
+* Established clear expectations and acknowledging diverse work cultures to build trust.
+* Leveraged digital tools and adapting communication methodologies to maintain productivity and team unity.
+
+These product managers discovered, quite unexpectedly, that remote work — originally a necessity due to the crisis — offered unexpected advantages such as increased deep work and adaptability. This led to conversations about the sustained incorporation of remote work practices within product management. Such insights from the case study offer reassurance that thriving in a remote environment is entirely feasible for product managers.
+
+------
+
+------
+
+One of the defining qualities of a good product manager is the ability to develop and envision a client-centered product. We present a case study that underscores the significance of this skill in the realm of product management.
+
+#### Case Study 1 - Client-Centred Approach:
+
+
+Jeff Bezos, the founder and former CEO of Amazon, is renowned for his unique leadership style that has been integral to creating the world's largest ecommerce and cloud computing platform. This case study explores the core tenets of Bezos’ approach, which have been pivotal in driving Amazon's consistent results over more than two decades​​.
+
+Bezos laid the foundation for Amazon's business and management methodology in his celebrated 1997 letter to shareholders, influencing Amazon's operations and culture to this day​​. In this letter he outlines Amazon's values, among which we can see:
+* Customer Obsession: Prioritizing customer needs over competitors
```

Comment:
I think this client-centred approach can be really useful for CSC301 students, especially for those who are working with a partner. It's important to always keep the partner's needs in mind and prioritize what they're looking for over what we might think is important, since we may not actually be familiar with their target audience and overall domain. The idea of consistently prioritizing your customer and their needs is a great takeaway that can be applied to not only CSC301 and the project, but also at future software engineering roles at companies of all sizes!